### DISCUSSION TITLE
VS2022 - "Rename" and "Find and Replace" don't work when Copilot is enabled

### DISCUSSION THREAD

**Discussion by chipnertkj** [community/community#14507] [Copilot Conversations]
**Description:**
Copilot prevents both Rename and Find and Replace from functioning properly in VS2022 when enabled. The search fails to find the specified keywords. This is only the case when the search scope is set to either Current project or Current solution. Disabling the extension shows that it is indeed an issue caused by Copilot.
This is not just a problem with replacing, searching on its own fails as well.

**Steps to reproduce:**
Open a VS2022 project with Copilot enabled and use either Rename or Find and Replace with search scope set to Current project or Current solution. 

**Expected behavior:**
Find and Replace and Rename should correctly find and replace keywords based on the input regardless of the scope set.

**I used:**
Visual Studio Community 2022 (64-bit), Version 17.1.2
Copilot version 1.10.0.3

**Comment by kzu** [comment]
It would be great if you could include a video or some screenshots showing the result you're getting. We're investigating the issue but we could not yet reproduce it. Are you getting any error hint in the Copilot Output Window? Thanks!
